### PR TITLE
Bug/oo #64 abos created today are inactiv

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: src/test/scala/ch/openolitor/stammdaten/calculateAktivTest.scala [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      IAbo.calculateAktiv(startsAtMidnight.toLocalDate, None) === true
+    }
+  }
+  "An abo starting today before midday and with no end date" should {
+    "be active" in {
+      val startsBeforeMidday = LocalDateTime.now.withTime(11, 59, 59, 999)
+      IAbo.calculateAktiv(startsBeforeMidday.toLocalDate, None) === true
+    }
+  }
+  "An abo starting today before midday and with no end date" should {
```
Comment (fabianpage) [inline src/test/scala/ch/openolitor/stammdaten/calculateAktivTest.scala]: wrong title of the test.

File: src/test/scala/ch/openolitor/stammdaten/calculateAktivTest.scala [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    "be active" in {
+      val startsAfterMidday = LocalDateTime.now.withTime(12, 0, 0, 1)
+      IAbo.calculateAktiv(startsAfterMidday.toLocalDate, None) === true
+    }
+  }
+  "An abo starting today before midday and with no end date" should {
```
Comment (fabianpage) [inline src/test/scala/ch/openolitor/stammdaten/calculateAktivTest.scala]: wrong title of the test.

**Review by fabianpage — APPROVED** [review]

File: src/test/scala/ch/openolitor/stammdaten/calculateAktivTest.scala (hunk 2 above)
Comment (mcmpp) [reply src/test/scala/ch/openolitor/stammdaten/calculateAktivTest.scala]: done

File: src/test/scala/ch/openolitor/stammdaten/calculateAktivTest.scala (hunk 1 above)
Comment (mcmpp) [reply src/test/scala/ch/openolitor/stammdaten/calculateAktivTest.scala]: done

**Pull request merged**